caas/kubernetes/provider/rbac.go

```diff
@@ -482,6 +499,9 @@ func (k *kubernetesClient) ensureRoleBinding(rb *rbacv1.RoleBinding) (out *rbacv
 		if v.GetName() == rb.GetName() {
 			name := v.GetName()
 			UID := v.GetUID()
+			if isRoleBindingEqual(v, *rb) {
```

Comment:
Do we really need to list them (above) based on labels? Would it be more efficient to get any existing rb by name and update if found, or create if not?

Reply:
I don't see a huge difference between list and get here.
I just had a check, by changing this, I need to update lots tests' gomock which is painful.

Reply:
ok, I changed my mind.